### PR TITLE
Added missing optional parameter FileChooserOptions

### PR DESCRIPTION
**Pull request by akshaydwivedi29**
Added optional parameter 'options' to support mime types

### CODE REVIEW

File: src/@ionic-native/plugins/file-chooser/index.ts [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
+    mime: string;
+}
 /**
  * @name File Chooser
  * @description
  *
  * Opens the file picker on Android for the user to select a file, returns a file URI.
  *
  * @usage
- * ```typescript
```
Comment (danielsogl) [inline src/@ionic-native/plugins/file-chooser/index.ts]: Remove that change

File: src/@ionic-native/plugins/file-chooser/index.ts [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
+ * typescript
  * import { FileChooser } from '@ionic-native/file-chooser/ngx';
  *
  * constructor(private fileChooser: FileChooser) { }
  *
  * ...
- *
- * this.fileChooser.open()
+ * let options :FileChooserOptions = {
+     mime:'application/pdf'
```
Comment (danielsogl) [inline src/@ionic-native/plugins/file-chooser/index.ts]: Fix the format here

File: src/@ionic-native/plugins/file-chooser/index.ts [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+    };
+ * this.fileChooser.open(options)
  *   .then(uri => console.log(uri))
  *   .catch(e => console.log(e));
  *
- * ```
```
Comment (danielsogl) [inline src/@ionic-native/plugins/file-chooser/index.ts]: Remove that change

**Comment by akshaydwivedi29** [comment]
can we reject this pr? i have created another one with proper formatting

**Pull request closed without merging**